pandas/_libs/tslibs/timezones.pyx

```diff
@@ -385,24 +385,26 @@ def tz_standardize(tz: tzinfo) -> tzinfo:
     -------
     tzinfo
 
-    Examples:
+    Examples
     --------
+    >>> from datetime import datetime
+    >>> from pytz import timezone
+    >>> tz = timezone('US/Pacific').normalize(datetime(2014,1,1, \
+    tzinfo=pytz.utc)).tzinfo
```

Comment:
you can use `...` to split this over multiple lines, would prefer that to `\`. Something like:

```suggestion
    >>> tz = timezone('US/Pacific').normalize(
    ... datetime(2014, 1, 1, tzinfo=pytz.utc)
    ... ).tzinfo
```